liberty-ls/src/main/java/io/openliberty/tools/langserver/utils/ServerPropertyValues.java

```diff
@@ -26,24 +26,24 @@ public class ServerPropertyValues {
     private final static List<String> YES_NO_VALUES = Arrays.asList("y", "n");
     
     private static HashMap<String, List<String>> validServerValues = new HashMap<String, List<String>>() {{
-        put("WLP_DEBUG_SUSPEND", YES_NO_VALUES);
-        put("WLP_DEBUG_REMOTE", YES_NO_VALUES);
+        put("WLP_DEBUG_SUSPEND", YES_NO_VALUES); // default yes
+        put("WLP_DEBUG_REMOTE", YES_NO_VALUES); // default undefined/disabled, so it's pointless selecting no
         
-        put("WLP_LOGGING_CONSOLE_FORMAT", Arrays.asList("dev", "simple", "json"));
-        put("WLP_LOGGING_CONSOLE_LOGLEVEL", Arrays.asList("INFO", "AUDIT", "WARNING", "ERROR", "OFF"));
-        put("WLP_LOGGING_CONSOLE_SOURCE", LOGGING_SOURCE_VALUES);
-        put("WLP_LOGGING_MESSAGE_FORMAT", Arrays.asList("simple", "json"));
-        put("WLP_LOGGING_MESSAGE_SOURCE", LOGGING_SOURCE_VALUES);
-        put("WLP_LOGGING_APPS_WRITE_JSON", BOOLEAN_VALUES);
+        put("WLP_LOGGING_CONSOLE_FORMAT", Arrays.asList("DEV", "SIMPLE", "JSON", "TBASIC")); // default "DEV"
+        put("WLP_LOGGING_CONSOLE_LOGLEVEL", Arrays.asList("AUDIT", "INFO", "WARNING", "ERROR", "OFF")); // default "AUDIT"
+        put("WLP_LOGGING_CONSOLE_SOURCE", LOGGING_SOURCE_VALUES); //default "message"
+        put("WLP_LOGGING_MESSAGE_FORMAT", Arrays.asList("SIMPLE", "JSON", "TBASIC")); // default "SIMPLE"
+        put("WLP_LOGGING_MESSAGE_SOURCE", LOGGING_SOURCE_VALUES); //default "message"
+        put("WLP_LOGGING_APPS_WRITE_JSON", BOOLEAN_VALUES); //unknown default
```

Comment:
This doesn't  make sense to me. A boolean property has to have a default. If this property is not specified, the server has a default behavior. What is it?